README.md

```diff
@@ -54,6 +58,7 @@ Having a bug tracking system doesn't make bug handling efficient and CI/CD doesn
 - What's the product/service schedule? (n-weekly releases / continuous deployment / multiple release streams / ...)
 - What happens after production incidents? Is there a culture of blameless analysis?
 - What are some ongoing challenges the team is experiencing that you are yet to resolve?
+- Do you employ any agile methodology? Is it scrum/kanban/waterfall?
```

Comment:
Maybe also add: Do teams have a scrum master or person in a similar role?

Reply:
Good point, gotta change it :)